conans/client/installer.py

```diff
@@ -226,7 +233,8 @@ def _download_bulk(self, install_order):
         if not downloads:
             return
 
-        ConanOutput().title("Downloading packages")
+        ConanOutput().title(f"Downloading {total_packages} package"
```

Comment:
``total_packages`` is not the right number for the download, only some of them will be downloaded, others might be in the cache, others might be built.

Reply:
Agh you're totally right, I had a brainfart moment